lib/rules/no-debug.ts

```diff
@@ -50,7 +51,7 @@ export default createTestingLibraryRule<Options, MessageIds>({
         }
         const initIdentifierNode = getDeepestIdentifierNode(node.init);
 
-        if (!initIdentifierNode) {
+        if (!initIdentifierNode || initIdentifierNode.name === 'console') {
```

Comment:
Not sure if we should check for `console` tbh, I think this should be handled more generically, no?

Reply:
What do you mean by "more generically"? `console` is a built-in object so we should be safe just checking that name in particular.

Reply:
I mean that we should probably only check if it's coming from TestingLibrary, no?

Reply:
That's already done inside `isDebugUtil`. However, if the Aggressive Reporting is enabled, it will assume any method called `debug` has to be reported, hence the necessity of ignoring those coming from the `console` object.

Reply:
right, that was my reasoning; I thought only the built-in should be excluded